src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangesRestClientTest.java

```diff
@@ -54,16 +54,16 @@
         return Iterables.transform(Arrays.asList(
                 queryParameter(
                         new TestQueryRequest().withQuery("is:open")
-                ).expectUrl("/changes/?q=is:open"),
+                ).expectUrl("/changes/?q=is%3Aopen"),
```

Comment:
Oh, I just wanted to know if you have tested queries which are encoded this way with a real Gerrit server REST API. ;)